eva-accession-deprecate/src/test/resources/logback.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
Configuration files don't need a copyright header as they aren't source code. This only makes it more convoluted.